### PR TITLE
[Term Entry] isprintable()

### PR DESCRIPTION
**Pull request by roberanegussie**
### Description
I have added a new term entry for the string class by creating a new file isprintable() which returns “True” if all characters in the string are printable or the string is empty else it returns “False”.

Closes Issue #3281 

### Type of Change

- Adding a new entry
### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/strings/terms/isprintable/isprintable.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  - 'paths/analyze-data-with-python'
+---
+
+The **`.isprintable()`** is a string method which returns a boolean value True if all charachters in the string are printable or if it's empty and False if any charachter in the string is nonprintable.
+
+## Syntax
+
+```pseudo
+str = "Hello World"
+print(str.isprintable())
```
Comment (SSwiniarski) [inline content/python/concepts/strings/terms/isprintable/isprintable.md:22]:
```suggestion
str.isprintable()
```

File: content/python/concepts/strings/terms/isprintable/isprintable.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+```
+
+This method is called on the String `str`, whose charachters are all printable and thus returning True.
+
+## Example
+
+In the following example, `.isprintable()` is used to find out if all charachters in the strings are printable.
```
Comment (SSwiniarski) [inline content/python/concepts/strings/terms/isprintable/isprintable.md:29]:
```suggestion
In the following example, `.isprintable()` is used to find out if all characters in the strings are printable.
```

**Comment by roberanegussie** [comment]
@SSwiniarski I have made the changes requested please review.

**Review by SSwiniarski — APPROVED** [review]
@roberanegussie I made one small edit and it looks good for a second review.

**Review by yangc95 — APPROVED** [review]

File: content/python/concepts/strings/terms/isprintable/isprintable.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Tags:
+  - 'String'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/analyze-data-with-python'
+---
+
+The **`.isprintable()`** is a string method that returns a boolean value `True` if all characters in the string are printable or if it's empty and `False` if any character in the string is nonprintable.
```
Comment (yangc95) [inline content/python/concepts/strings/terms/isprintable/isprintable.md:16]:
```suggestion
**`.isprintable()`** is a string method that returns a boolean value `True` if all characters in the string are printable or if it's empty, and `False` if any character in the string is non-printable.
```

**Pull request merged**